### PR TITLE
Laser menu tweaks; Changed some default colors

### LINKED ISSUE #5841: Default colors for laser doors and laser freeze
image

I hadn't thought about it at all. We talked about the default laser color for shotgun, and I like it. The default laser color for the rifle shouldn't change, probably.

Are the default colors for doors (turquoise) and freeze (red) a good choice?

I think the red for freeze doesn't look like something that will freeze you, red is usually not used for freeze. Grey would probably make it hard to see though.

Might be a good chance to find good defaults before it ships in a release.

### PR DESCRIPTION
**Pull request by VoxelDoesCode**
fixes #5841 

image

## Checklist

- [x] Tested the change ingame
- [x] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [x] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Review by def- — APPROVED** [review]

**Comment by heinrich5991** [comment]
Looks good to me. clang-formatted.

**Comment by VoxelDoesCode** [comment]
I still want to add a Reset all button at the bottom, if that's fine.

**Comment by heinrich5991** [comment]
Sorry, should've asked first. Reverted to the original state.

File: src/game/client/components/menus_settings.cpp [hunk 1]
```diff
@@ -2990,10 +2992,39 @@ void CMenus::RenderSettingsAppearance(CUIRect MainView)
...
+			g_Config.m_ClLaserDoorOutlineColor = g_Config.m_ClLaserRifleOutlineColor;
+			g_Config.m_ClLaserDoorInnerColor = g_Config.m_ClLaserRifleInnerColor;
+			g_Config.m_ClLaserFreezeOutlineColor = g_Config.m_ClLaserRifleOutlineColor;
+			g_Config.m_ClLaserFreezeInnerColor = g_Config.m_ClLaserRifleInnerColor;
+		}
+
+		// values taken from the CL commands
+		LeftView.HSplitTop(10.0f, 0x0, &LeftView);
+		LeftView.HSplitTop(20.0f, &Button, &LeftView);
+		if(DoButton_Menu(&s_AllToDefaultResetID, Localize("Reset all"), 0, &Button))
```
Comment (def-) [inline src/game/client/components/menus_settings.cpp:3017]: Use "Reset to defaults", then we don't need a new translation.

**Review by def- — CHANGES_REQUESTED** [review]

**Pull request merged**